src/libraries/System.Private.CoreLib/src/System/IO/FileStream.cs

```diff
@@ -130,6 +130,43 @@ public FileStream(string path, FileMode mode, FileAccess access, FileShare share
         }
 
         public FileStream(string path, FileMode mode, FileAccess access, FileShare share, int bufferSize, FileOptions options)
+            : this(path, mode, access, share, bufferSize, options, 0)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="T:System.IO.FileStream" /> class with the specified path, creation mode, read/write and sharing permission, the access other FileStreams can have to the same file, the buffer size,  additional file options and the allocation size.
+        /// </summary>
+        /// <param name="path">A relative or absolute path for the file that the current <see langword="FileStream" /> object will encapsulate.</param>
+        /// <param name="mode">One of the enumeration values that determines how to open or create the file.</param>
+        /// <param name="access">A bitwise combination of the enumeration values that determines how the file can be accessed by the <see langword="FileStream" /> object. This also determines the values returned by the <see cref="P:System.IO.FileStream.CanRead" /> and <see cref="P:System.IO.FileStream.CanWrite" /> properties of the <see langword="FileStream" /> object. <see cref="P:System.IO.FileStream.CanSeek" /> is <see langword="true" /> if <paramref name="path" /> specifies a disk file.</param>
+        /// <param name="share">A bitwise combination of the enumeration values that determines how the file will be shared by processes. The default value is Read.</param>
+        /// <param name="bufferSize">A positive <see cref="T:System.Int32" /> value greater than 0 indicating the buffer size. The default buffer size is 4096.</param>
+        /// <param name="options">A bitwise combination of the enumeration values that specifies additional file options. The default value is None which means synchronous IO.</param>
+        /// <param name="allocationSize">The initial allocation size in bytes for the file. A value has no effect unless it's positive and a regular file is being created, overwritten, or replaced.</param>
+        /// <exception cref="T:System.ArgumentNullException"><paramref name = "path" /> is <see langword="null" />.</exception>
+        /// <exception cref="T:System.ArgumentException"><paramref name="path" /> is an empty string (""), contains only white space, or contains one or more invalid characters.
+        /// -or-
+        /// <paramref name="path" /> refers to a non-file device, such as "con:", "com1:", "lpt1:", etc. in an NTFS environment.</exception>
```

Comment:
In normal xml strings (not markdown), you can wrap in `<c></c>` as a substitute for backticks (\` \`):
```suggestion
        /// <paramref name="path" /> refers to a non-file device, such as <c>CON:</c>, <c>COM1:</c>, <c>LPT1:</c>, etc. in an NTFS environment.</exception>
```